controllers/fenceagentsremediation_controller_test.go

```diff
@@ -0,0 +1,214 @@
+/*
+Copyright 2023.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/medik8s/fence-agents-remediation/api/v1alpha1"
+	"github.com/medik8s/fence-agents-remediation/pkg/cli"
+)
+
+const (
+	defaultNamespace = "default"
+)
+
+var (
+	executedCommand []string
```

Comment:
I agree.

I stored the _**executedCommand**_ in the [mock executor field](https://github.com/medik8s/fence-agents-remediation/pull/11/files#diff-fac0cf56dda42683c1e24b46c87cfa4cca46c444145ee69de9e264db50f10289R198), and I am not sure how to call the _cliCommandsEquality_ function in a way that use the same mock executer (otherwise, I will have a pointer to mockExecuter but without the _executedCommand_ value).
Is there any similar example I can follow?

Reply:
No more global vars over there, thanks to @mshitrit  🙏🏻